docs/sources/administration/correlations/correlation-permissions/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Permissions
+weight: 20
+---
+
+# Permissions
+
+Users with [Viewer base role]({{< relref "/docs/grafana/latest/administration/roles-and-permissions/" >}}) or with [datasources:query RBAC role]({{< relref "/docs/grafana/latest/administration/roles-and-permissions/access-control/" >}}) can:
+
+- Use correlations in Explore’s visualizations
+- List all available correlations in read-only mode
+
+Users with [Admin base role]({{< relref "/docs/grafana/latest/administration/roles-and-permissions/" >}}) or with [datasources:write RBAC role]({{< relref "/docs/grafana/latest/administration/roles-and-permissions/access-control/" >}}) can:
+
+- User with Admin base role or with datasources:write RBAC role can:
```

Comment:
is this a leftover?

Reply:
Removed in d28cb26e1b34fd1f5952e8efc46b510a1d48e6eb